src/account-info/LogoutButton.js

```diff
@@ -8,6 +8,7 @@ import boundActions from '../boundActions';
 import { ZulipButton } from '../common';
 import unregisterPush from '../api/unregisterPush';
 import { getAuth } from '../selectors';
+import { logErrorToSentry } from '../utils/sentry';
```

Comment:
We want to make this logging more abstract. We do not need to include Sentry's name here.
`logErrorRemotely` or even just `logError` is good enough. And it will be in `utils/logging`.